_posts/2017-04-06-git-aliases-ideas-from-our-setups.md

```diff
@@ -0,0 +1,71 @@
+---
+date: 2017-04-07
+title: "Git rich or die tryin' 01: Ideas from our setups"
+categories:
+  - CLI
+  - Git
+  - Alias
+author_staff_member: 02_fausto
+---
+
+This post contains a list of the Git aliases we use the most, particularly the ones we find interesting (i.e., not `c = commit`) and that have increased our productivity using Git.
```

Comment:
This post is part of a series about our Git setups. In this series we will focus on the Git aliases we use the most, ...
A complete compilation can be found here link_to_never_forgit